src/app/pages/installation/_mac_designer.twig

```diff
@@ -10,6 +10,11 @@ Use the Axon Ivy Designer <a href="/download/sprint">9.2</a> or newer to work wi
 
 {% else %}
 <h3>Mac Designer for BigSur and older</h3>
+<p style="background-color:orange; padding:20px;">
+This Axon Ivy Designer runs best on Macs with Intel Silicon (x64 architecture).</br>
```

Comment:
I only saw the term silicon with the combination of apple chips. I'm not sure if this is also used for intel...